packages/js-dpp/lib/test/fixtures/getIdentityUpdateTransitionFixture.js

```diff
@@ -0,0 +1,30 @@
+const protocolVersion = require('../../version/protocolVersion');
+const stateTransitionTypes = require('../../stateTransition/stateTransitionTypes');
+const getInstantAssetLockProofFixture = require('./getInstantAssetLockProofFixture');
+const generateRandomIdentifier = require('../utils/generateRandomIdentifier');
+const IdentityUpdateTransition = require('../../identity/stateTransition/IdentityUpdateTransition/IdentityUpdateTransition');
+const IdentityPublicKey = require('../../identity/IdentityPublicKey');
+
+module.exports = function getIdentityUpdateTransitionFixture() {
+  const rawStateTransition = {
+    protocolVersion: protocolVersion.latestVersion,
+    type: stateTransitionTypes.IDENTITY_UPDATE,
+    assetLockProof: getInstantAssetLockProofFixture().toObject(),
+    identityId: generateRandomIdentifier(),
+    revision: 0,
+    addPublicKeys: [
+      {
+        id: 3,
+        type: IdentityPublicKey.TYPES.ECDSA_SECP256K1,
+        data: Buffer.from('AkVuTKyF3YgKLAQlLEtaUL2HTditwGILfWUVqjzYnIgH', 'base64'),
+        purpose: IdentityPublicKey.PURPOSES.AUTHENTICATION,
+        securityLevel: IdentityPublicKey.SECURITY_LEVELS.MASTER,
```

Comment:
We shouldn't be able to send with `disabledAt`. It must be changed only programmatically.